### ISSUE TITLE
Windows10 init.elc file-error "Renaming" "Permission denied"

### ISSUE THREAD

**Issue by ProfessorX** [bbatsov/prelude#1082]
This is not showing every time I start Emacs on Windows 10 15063. But it is annoying.

The "hotfix" is to go to the .emacs.d folder and delete all init.elc, init.elc~ as well as the temp files generated by Emacs. After that restart Emacs it's fine.

I have given the EMACS@DESKTOP~XXXXXX all rights and **now I am even running it as admin**, it still recurs.

Debugger output as follows (mostly)

`
Debugger entered--Lisp error: (file-error "Renaming" "Permission denied" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/tmp9152UxR" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/init.elc")
  rename-file("c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/tmp9152UxR" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/init.elc" t)
  basic-save-buffer-2()
  basic-save-buffer-1()
  basic-save-buffer(nil)
  save-buffer()
  package--ensure-init-file()
  package-initialize()
  byte-code(("\304\305!\210\304\306!\210\307=\203
`

A normal start would be as follows in Message:
`
Prelude is powering up... Be patient, Master YANAN XIAO!
Loading personal configuration files in c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/personal/preload...
Loading Prelude’s core...
ad-handle-definition: ‘ido-completing-read’ got redefined [2 times]
Loading c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/savefile/recentf...done
Loading Prelude’s modules...
Loading c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/prelude-modules.el (source)...done
Loading personal configuration files in c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/personal...
Loading c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/personal/custom.el (source)...done
Prelude is ready to do thy bidding, Master YANAN XIAO!
For information about GNU Emacs and the GNU system, type C-h C-a.
Prelude tip: Press <C-c y> to search in YouTube.
Updating buffer list...
Formats have changed, recompiling...done
Updating buffer list...done
Commands: m, u, t, RET, g, k, S, D, Q; q to quit; h for help
`

**Comment by ProfessorX** [comment]
> Debugger entered--Lisp error: (file-error "Renaming" "Permission denied" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/tmp13548-cH" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/init.elc")
  rename-file("c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/tmp13548-cH" "c:/Users/YANAN XIAO/AppData/Roaming/.emacs.d/init.elc" t)
  basic-save-buffer-2()
  basic-save-buffer-1()
  basic-save-buffer(nil)
  save-buffer()
  package--ensure-init-file()
  package-initialize()
  byte-code("\304\305!\210\304\306!\210\307=\203 

HOW TO RE-PRODUCE

I did `package-list-packages` and updated all the listed possible updates.

Then I use `prelude-update` to update prelude.

Then I restart Emacs as noted. It happened again. 

Usually if I don't update nothing like this will happen. I will try avoiding updaing prelude next time and see what will happen.

**Comment by ProfessorX** [comment]
**Update**

Issue solved.

I was using Ubuntu 16.04LTS on this Surface Pro 3 computer. Later I would like to play **Hearthstone** on it when I am on business trips, so I can avoid going to clubs for fun.

I then copied the `.emacs.d` folder into my USB flash drive. When I **reinstalled** the Windows 10 on this computer I **simply** copied the `.emacs.d` folder into my USER/Roaming stuff.

Today when I did the `prelude-update` the same crap happened, I went to check the **directory** rights (UAC in Linux? Long time for these terms.)

IT WAS PARTIALLY **READ ONLY**. FROM THIS MOMENT I KNOW I FUCKED UP. (So it's a good habit to check these access control stuff when copying files, even under Linux as I can remember, using cp -xxx).

**After I uncheck the *READ ONLY* property, everything works as smooth as Black Girls' skin.**
